tools/VersionController/Models/VersionFileHelper.cs

```diff
@@ -65,11 +66,11 @@ public VersionFileHelper(string rootDirectory, string outputModuleManifestPath,
         public string ChangeLogPath => Directory.GetFiles(ProjectDirectory, "ChangeLog.md").FirstOrDefault();
 
         public List<string> AssemblyInfoPaths => Directory.GetFiles(SrcDirectory, "AssemblyInfo.cs", SearchOption.AllDirectories)
-                                                            .Where(f => !f.Contains("Stack") && !f.Contains(".Test"))
+                                                            .Where(f => (!f.Contains("Stack") || WhiteList.Contains(f)) && !f.Contains(".Test"))
                                                             .ToList();
 
-        public string GalleryModuleDirectory => Path.Combine(OutputModuleDirectory, ModuleName);
```

Comment:
Please verify these two logic with:
1. Debug in Vistual Studio.
1. Run `RunVersionController.ps1` under both directory `azure-powershell\tools` and `azure-powershell` with two parameter sets:
    1. `RunVersionController.ps1 -Release 2020-03-March`
    1. `RunVersionController.ps1 -ModuleName az.accounts`